Automattic/woocommerce-payments
# Redirect to onboarding when account disconnected

Fixes #648

#### Changes proposed in this Pull Request

Redirect to the onboarding page when the WooCommerce Payments account is disconnected.

| Before | After |
|-|-|
|<img width="1071" alt="Screen Shot 2021-06-21 at 10 16 28 am" src="https://user-images.githubusercontent.com/1527164/122692706-049a5b80-d27a-11eb-98bc-bcb47b7d5563.png">|<img width="1071" alt="Screen Shot 2021-06-21 at 10 17 03 am" src="https://user-images.githubusercontent.com/1527164/122692714-0ebc5a00-d27a-11eb-8db4-c183144e7bf4.png">|

#### Testing instructions

* Start with a new site, or disconnect your site from WCPay/Stripe.
* Open this page: `http://<YOUR_SITE>/wp-admin/admin.php?page=wc-admin&path=/payments/deposits`
* Observe you are redirected to `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect`

-------------------
